src/AppPrintOnly.test.tsx

```diff
@@ -59,7 +85,17 @@ beforeEach(() => {
 it('VxPrintOnly flow', async () => {
   jest.useFakeTimers()
 
-  const { getByText } = render(<App />)
+  const { getAllByText, getByLabelText, getByText, getByTestId } = render(
+    <App />
+  )
+  const getAllByTextWithMarkup = (text: string) =>
```

Comment:
I'd add a comment explaining what this is doing and possibly also change the name. At least link to your SO post.